src/components/comments/postComment.jsx

```diff
@@ -55,6 +61,7 @@ function CommentPublication(props) {
                 id="comment"
                 className="block w-full my-2 border-0 focus:ring-0 sm:text-sm"
                 placeholder="Add your comment..."
+                value={message}
```

Comment:
Why do you put the message here ? This thumb is for when you upload a file, to have a preview of it